coalib/results/Diff.py

```diff
@@ -438,9 +436,15 @@ def change_line(self, line_nr, original_line, replacement):
                                                replacement)
             replacement = ''.join((orig_diff + new_diff).modified)
 
-        linediff.change = (original_line, replacement)
+        linediff.change = (self._file[line_nr-1], replacement)
         self._changes[line_nr] = linediff
 
+    def change_line(self, line_nr, original_line, replacement):
+        logging.debug('Use of change_line method is deprecated. Instead '
```

Comment:
Is `debug` the best idea? We use `warn` [here](https://github.com/coala/coala/pull/3546/commits/99c7b52106879dc57a0bdd485578fc3f55a0845c#diff-e3b01f8046c9fb1b13110e5b3acc07abR123) and this is similar - only bears use it. If that warrants a warning, maybe this is worth a warning too. Others?